include/dav1d/headers.rs

```diff
@@ -2096,6 +2096,48 @@ pub struct Dav1dFrameHeader_segmentation {
     pub qidx: [u8; DAV1D_MAX_SEGMENTS as usize],
 }
 
+/// Like a `[bool; RAV1D_MAX_SEGMENTS as usize]`,
+/// but the [`bool`]s are compressed into the bits of the [`u8`].
+#[derive(Clone, Copy, Default)]
+pub struct LossLess {
```

Comment:
Nit: This should be `Lossless` (one word), instead of "Loss Less" (two words).

Reply:
But `LossLess` looks cooler 🥺.